### PR TITLE
feat: command for running VMAF on arbitrary files using configured pipeline

### PR DESCRIPTION
**Pull request by grusell**
* feat: command for running VMAF on arbitrary files using configured pipeline

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -307,6 +309,25 @@ Options:
 If your job uses variables ([See above](#using-variables-in-the-job-definition)), the variables that should
 be included in the csv data should be specified with the `--variables` option.
 
+### Run VMAF only
+
+This command allows using a configured pipeline to run VMAF measurements on any files without an related autovmaf job.
```
Comment (oshinongit) [inline README.md:314]: Tiny spelling error should be "a related autovmaf job"

File: src/cli.ts [hunk 2]
```diff
@@ -133,10 +138,57 @@ async function run() {
...
+    )
     .help()
     .parse();
 }
 
+async function runVmaf(argv) {
+  const reference = argv.reference;
+  const distorted = argv.distorted;
+  const pipeline = await loadPipeline(argv.pipeline);
+  const qualityFile = argv.distorted.replace('.mp4', '_vmaf.json');
```
Comment (oshinongit) [inline src/cli.ts:178]: Will we only allow running on mp4 files?

Comment (grusell) [reply src/cli.ts:178]: Not really this was copy pasted from other part of the code. Will fix.

File: README.md (hunk 1 above)
Comment (grusell) [reply README.md]: fixed.

File: src/cli.ts (hunk 2 above)
Comment (grusell) [reply src/cli.ts]: Fixed.

**Pull request merged**